fhir-server-test/src/test/java/com/ibm/watson/health/fhir/server/test/ExportOperationTest.java

```diff
@@ -47,7 +47,7 @@
 
     public static final String FORMAT = "application/fhir+ndjson";
 
-    public static final boolean ON = true;
+    public static final boolean ON = false;
```

Comment:
what is ON?

Reply:
ON constant was my way of not overriding DEBUG in an integration test.  I don't think it will pass otherwise. *it flips the test to ON or OFF.